ckan/adx_config.ini

```diff
@@ -177,7 +177,7 @@ ckanext.validation.default_validation_options = {"limit_rows": 50000, "limit_err
 ckanext.unaids.auth0_domain =
 ckanext.unaids.oauth2_api_audience =
 ckanext.unaids.oauth2_required_scope = access:adr
-ckanext.unaids.profile_editor_url = http://profile-editor.minikube
+ckanext.unaids.profile_editor_url = http://localhost:3000
```

Comment:
Presuming this is a small fix for a matter unrelated to the maptiles, that you just want to resolve at the same time?

Reply:
OK you have removed it :+1:

Reply:
It was a mistake on my end. I've removed those extra changes.